### DISCUSSION TITLE
UI suggestions for the iOS app

### DISCUSSION THREAD

**Discussion by dpad85** [ACINQ/phoenix#202] [Improving Phoenix]
This post centralizes a few suggestions that have been made regarding Phoenix UI. Thanks again to the contributors.

From @Bosch-0: https://www.figma.com/file/Eph6YJSCSKQJnHxT9vCfmI/Phoenix?node-id=1%3A120

From @pavlenex: https://www.figma.com/file/uaAarmZzZBg9J1HaHaOcCX/Phonenix-Lightning-Wallet?node-id=1%3A40

**Comment by pavlenex** [comment]
Hey @dpad85 thanks for following along, was planning to open a few issues with more concrete suggestions will post here tomorrow.

**Reply by pavlenex** [reply]
Hey @dpad85 since we've opened quite a few issues, please understand it's not our goal to spam or overwhelm you. If you feel that any of these issue isn't relevant to you guys, feel free to close them, though for us it would be helpful to know why you've decided against certain recommendation. I understand that it's sometimes hard to understand certain decisions from our angle, and you guys know your users better than we do.

Since we've opened up quite a few issues, please let me know if we should keep doing that or should we first verify if we should opened them by posting here or through some other channel. 

- https://github.com/ACINQ/phoenix/issues/184
- https://github.com/ACINQ/phoenix/issues/196
- https://github.com/ACINQ/phoenix/issues/197
- https://github.com/ACINQ/phoenix/issues/198
- https://github.com/ACINQ/phoenix/issues/199
- https://github.com/ACINQ/phoenix/issues/200
- https://github.com/ACINQ/phoenix/issues/204
- https://github.com/ACINQ/phoenix/issues/205

Awesome to see https://github.com/ACINQ/phoenix/issues/199 already merged in and it seems that #184 also has a PR 🔥

**Reply by dpad85** [reply]
Opening issues is fine, no worries! They are pretty well defined and I agree with most of them

**Comment by Bosch-0** [comment]
For those who aren't Figma users here is what I shared on Telegram originally regarding some on boarding changes. Note the new onboarding only shows creating a wallet wallet not importing an existing one. 

![image](https://user-images.githubusercontent.com/55287964/136886626-4544c51a-ef26-4476-817e-80fdfaa3ae9a.png)

> Hey guys, I'm a UX designer who focuses on Bitcoin applications. Myself and others over at the Bitcoin Design Community (https://bitcoin.design/) recently did a review of Phoenix's UX as we begin to integrate Lightning focus content and focus our Design Guide project on Mobile first, non-custodial, Bitcoin Lightning products like Phoenix - so you could say we are big fans! 
> 
> This was a small review on the onboarding section I did with a new onboarding mockup integrating the suggestions I made. Would love to get some feedback from the community / talk to devs if this is something you'd want to implement. 
> 
> I have feedback on other areas of the app but I have not finalized them / want to play around a little more, but if you are interested to see the WIP here is the Figma file: https://www.figma.com/file/Eph6YJSCSKQJnHxT9vCfmI/Phoenix?node-id=1%3A120

**Reply by Bosch-0** [reply]
One minor change, 'Restore wallet' button should be renamed to 'Restore Phoenix wallet' as you can only restore Phoenix wallets within Phoenix. This helps further solidify that Phoenix seeds are unique and not interoperable.
